jina/flow/base.py

```diff
@@ -1627,6 +1631,26 @@ def workspace_id(self, value: str):
                         for i in v:
                             i.workspace_id = value
 
+    @property
+    def env(self) -> Optional[Dict]:
+        """Get all envs to be set in the Flow
+
+        :return: envs as dict
+        """
+        return self.args.env
+
+    @env.setter
+    def env(self, value: Dict[str, str]):
+        """set env vars for flow & all pods.
+        This can be used by jinad to set envs for Flow and all child objects
+
+        :param value: value to be set
+        """
+        if isinstance(value, dict):
```

Comment:
no need to set this `if`. If ur typehint suggests this there is no point to check.